contracts/StakeCurate.sol

```diff
@@ -197,53 +196,54 @@ contract StakeCurate is IArbitrable, IEvidence {
 
   /**
    * @dev Creates a list. They store all settings related to the dispute, stake, etc.
-   * @param _governor The address of the governor.
+   * @param _governorId The id of the governor.
    * @param _requiredStake The Cint32 version of the required stake per item.
    * @param _removalPeriod The amount of seconds an item needs to go through removal period to be removed.
    * @param _arbitratorExtraDataId Id of the internally stored arbitrator extra data
    * @param _metaEvidence IPFS uri of metaEvidence
    */
   function createList(
-    address _governor,
+    uint64 _governorId,
     uint32 _requiredStake,
     uint32 _removalPeriod,
-    uint32 _arbitratorExtraDataId,
+    uint64 _arbitratorExtraDataId,
     string calldata _metaEvidence
   ) external {
+    //require(accounts[_governorId].wallet == msg.sender, "Governor must exist and be the function caller");
     uint64 listId = listCount++;
     List storage list = lists[listId];
-    list.governor = _governor;
+    list.governorId = _governorId;
     list.requiredStake = _requiredStake;
     list.removalPeriod = _removalPeriod;
     list.arbitratorExtraDataId = _arbitratorExtraDataId;
-    emit ListCreated(_governor, _requiredStake, _removalPeriod, _arbitratorExtraDataId);
+    emit ListCreated(_governorId, _requiredStake, _removalPeriod, _arbitratorExtraDataId);
     emit MetaEvidence(listId, _metaEvidence);
   }
 
   /**
    * @dev Updates an existing list. Can only be called by its governor.
    * @param _listId Id of the list to be updated.
-   * @param _governor Address of the new governor.
+   * @param _governorId Id of the new governor.
    * @param _requiredStake Cint32 version of the new required stake per item.
    * @param _removalPeriod Seconds until item is considered removed after starting removal.
    * @param _arbitratorExtraDataId Id of the new arbitrator extra data
    * @param _metaEvidence IPFS uri of metaEvidence
    */
   function updateList(
     uint64 _listId,
-    address _governor,
+    uint64 _governorId,
     uint32 _requiredStake,
     uint32 _removalPeriod,
-    uint32 _arbitratorExtraDataId,
+    uint64 _arbitratorExtraDataId,
     string calldata _metaEvidence
   ) external {
     List storage list = lists[_listId];
-    require(list.governor == msg.sender, "Only governor can update list");
-    list.governor = _governor;
+    require(accounts[list.governorId].wallet == msg.sender, "Only governor can update list");
```

Comment:
That's not the criterion for existing
You've changed the functionality of `updateList`

Reply:
With that change you're allowing anyone to change the parameters of the list